connector/countconnector/go.mod

```diff
@@ -0,0 +1,55 @@
+module go.opentelemetry.io/collector/connector/countconnector
+
+go 1.19
```

Comment:
Before merging, can you ensure this is aligned with the main go.mod? I think we are still on 1.18

Reply:
Fixed